tests/crypto/crl_tests.c

```diff
@@ -139,22 +152,17 @@ static void _test_get_dates(void)
     oe_datetime_t next;
     OE_TEST(oe_crl_get_update_dates(&crl, &last, &next) == OE_OK);
 
-    OE_TEST(last.year == _time.year);
-    OE_TEST(last.month == _time.month);
-    OE_TEST(last.day == _time.day);
-    OE_TEST(last.hours == _time.hours);
-    OE_TEST(last.minutes == _time.minutes);
-    // TEMPORARILY Disabled: _time is the certificate file's timestamp
-    // whereas last.seconds is the actual issue time of the certificate.
-    // The two values are not guaranteed to be equal.
-    // OE_TEST(last.seconds == _time.seconds);
+    int32_t diff = _diff_oe_datetime(&last, &_time);
+    printf("=== Diff between last and current time: %d\n", diff);
+    OE_TEST(abs(diff) < 60); // Ensure times are within 60 seconds
```

Comment:
Suggest creating a #define'd constant that you can use here and in line 216 rather than hard coding 60 in both places.   And is the number 60 arbitrary or is there a reason it's better than say 10 or 300?

Reply:
It's arbitrary. I'm open to changing it, I wanted to experiment with what the value normally is, although it my local testing it's pretty much always 0 or 1